modules/core/server/controllers/mongoose-filtering.server.controller.js

```diff
@@ -0,0 +1,54 @@
+﻿// Service for managing Filtering/Parameterized-Queries.
+
+var _query = null;
+var _modifiedQuery = null;
+var _totalDocumentCount = null;
+var _page = 0;
+var _take = 50; // default to prevent unreasonable take() requests
```

Comment:
I think I should move the default, and cap (below) for `_take` responsibility to the collection controller. We may want have different defaults & caps on each collection. For instance, a messaging module might want only the last 50 messages returned, and Articles may want 50.